Frontend/src/components/Input/AutocompleteInput.svelte

```diff
@@ -42,9 +47,12 @@
     inputId={id}
     {noResultsText}
     {disabled}
+    localSorting={onlyNumbers}
     hideArrow={true}
+    sortByMatchedKeywords={true}
+    {itemSortFunction}
```

Comment:
@skjerns We shouldn't hardcode this function here since AutocompleteInput is a generic input component which can also be used for objects without ids. For example it is currently also used for postal codes and cities in the rental form. Please check if these still work. I think most of these new properties should also become properties of AutocompleteInput.

Reply:
Seems like a good idea! I implemented the proposed solution

Reply:
The structure looks good now :) I just added a few more comments.